src/viam/media/video.py

```diff
@@ -34,6 +36,25 @@ def close(self):
         """Close the image and release resources. For RawImage, this is a noop."""
         return
 
+    def bytes_to_depth_array(self) -> List[array]:
```

Comment:
I thought this was the best place to put this function since we'll be using it for `RawImage`. Thoughts?